### PR TITLE
Cleanup content that exists in Azure Architecture Center now

### PR DESCRIPTION
**Pull request by heoelri**
This PR removes the content that lives in Azure Architecture Center now. Not much content will be left in this repository besides the landing page and a couple of supporting artifacts.

**TODO**
* [ ] Cleanup RI repositories and update links if needed
* [ ] Remove abandones images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Azure Mission-Critical is an open source project that provides a **prescriptive
 
 ## Azure Mission-Critical | Navigation
 
-- [Introduction | What is Mission-Critical?](./docs/introduction/README.md) - Detailed introduction into Mission-Critical, the problem it is intended to solve and the value it can provide.
+- [Introduction | What is Mission-Critical?](https://docs.microsoft.com/azure/architecture/framework/mission-critical/mission-critical-overview) - Detailed introduction into Mission-Critical, the problem it is intended to solve and the value it can provide.
 
-- [Design Methodology | Mission-Critical Architectural Approach](./docs/design-methodology/README.md) - Prescriptive guidance aligned to 8 critical design areas guides users to design and build an mission-critical application, outlining a recommended decision process.
+- [Design Methodology | Mission-Critical Architectural Approach](https://docs.microsoft.com/azure/architecture/framework/mission-critical/mission-critical-design-methodology) - Prescriptive guidance aligned to 8 critical design areas guides users to design and build an mission-critical application, outlining a recommended decision process.
```
Comment (heoelri) [inline README.md:18]: Good catch, I missed that. We can then remove the `README.md` in `/docs/design-methodology`, too.

Comment (hjscherer) [reply README.md:18]: yes

**Comment by heoelri** [comment]
@hjscherer I've just noticed that the `Introduction.md` and other files are not linked on the main `README.md` anymore - all links on the main `README.md` are not either pointing to docs.microsoft.com or to one of the reference implementations. So we might be able to drop the content in the `/docs/` directory completely.

**Comment by hjscherer** [comment]
> @hjscherer Hansjoerg Scherer FTE I've just noticed that the `Introduction.md` and other files are not linked on the main `README.md` anymore - all links on the main `README.md` are not either pointing to docs.microsoft.com or to one of the reference implementations. So we might be able to drop the content in the `/docs/` directory completely.

I think there is still value of having this content in the repo. As an overview and directly point to specific design areas. How about now?

**Review by hjscherer — APPROVED** [review]
LGTM

**Comment by heoelri** [comment]
> > @hjscherer
> >             
> >             Hansjoerg Scherer
> >             FTE Hansjoerg Scherer FTE I've just noticed that the `Introduction.md` and other files are not linked on the main `README.md` anymore - all links on the main `README.md` are not either pointing to docs.microsoft.com or to one of the reference implementations. So we might be able to drop the content in the `/docs/` directory completely.
> 
> I think there is still value of having this content in the repo. As an overview and directly point to specific design areas. How about now?

Better, but now that you're refering to `/docs/README.md` the `/docs/Introduction.md` is still abandoned - so we might be able to drop the `/docs/Introduction.md` and keep only `README.md` and `design-areas.md`.

**Comment by heoelri** [comment]
> > > @hjscherer
> > >             
> > >             Hansjoerg Scherer
> > >             FTE
> > > ```
> > >         Hansjoerg Scherer
> > >         FTE Hansjoerg Scherer FTE I've just noticed that the `Introduction.md` and other files are not linked on the main `README.md` anymore - all links on the main `README.md` are not either pointing to docs.microsoft.com or to one of the reference implementations. So we might be able to drop the content in the `/docs/` directory completely.
> > > ```
> > 
> > 
> > I think there is still value of having this content in the repo. As an overview and directly point to specific design areas. How about now?
> 
> Better, but now that you're refering to `/docs/README.md` the `/docs/Introduction.md` is still abandoned - so we might be able to drop the `/docs/Introduction.md` and keep only `README.md` and `design-areas.md`.

Ah, wait - its still linked in `/docs/README.md` so I think its ok for now.

**Comment by hjscherer** [comment]
> > > > @hjscherer
> > > > ```
> > > >         Hansjoerg Scherer
> > > >         FTE
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > ```
> > > >         Hansjoerg Scherer
> > > >         FTE Hansjoerg Scherer FTE I've just noticed that the `Introduction.md` and other files are not linked on the main `README.md` anymore - all links on the main `README.md` are not either pointing to docs.microsoft.com or to one of the reference implementations. So we might be able to drop the content in the `/docs/` directory completely.
> > > > ```
> > > 
> > > 
> > > I think there is still value of having this content in the repo. As an overview and directly point to specific design areas. How about now?
> > 
> > 
> > Better, but now that you're refering to `/docs/README.md` the `/docs/Introduction.md` is still abandoned - so we might be able to drop the `/docs/Introduction.md` and keep only `README.md` and `design-areas.md`.
> 
> Ah, wait - its still linked in `/docs/README.md` so I think its ok for now.

yeah was about to write this back 👍

**Review by CalCof — APPROVED** [review]

**Pull request merged**